src/index.html

```diff
@@ -1,14 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="home-link">
+        <img
+          src="images/logo.png"
+          class="logo"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="apple" class="nav__link is-active">Apple</a>
```

Comment:
let that be an anchor link
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```